cric-predict/src/index.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';import React from 'react';
```

Comment:
you should be able to import bootstrap's components instead

Reply:
Now removed links from index.html and imported bootstrap components instead.